core/modules/compare/store/index.ts

```diff
@@ -2,10 +2,9 @@ import { Module } from 'vuex'
 import actions from './actions'
 import getters from './getters'
 import mutations from './mutations'
-import RootState from '../../types/RootState'
-import CompareState from './types/CompareState'
+import CompareState from '../types/CompareState'
```

Comment:
```suggestion
import CompareState from '../types/CompareState'
import RootState from '@vue-storefront/store/types/RootState'
```
you can use `RootState` instead of any

Reply:
done